.github/workflows/deploy.yaml

```
@@ -38,9 +38,9 @@ jobs:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
 
       - name: Set go environment
-        uses: actions/setup-go@v4
+        uses: actions/setup-go@v5
         with:
-          go-version: '1.20'
+          go-version: '1.22'
```

Could use version from `go.mod` similarly than in DevEx repos to avoid configuring it to multiple files.

```suggestion
          go-version-file: 'go.mod'
```

---

Thanks :+1:  Didn't know you could use that. I'll change that.